OWNERS

```diff
@@ -0,0 +1,13 @@
+# See Config struct in https://github.com/kubernetes/test-infra/blob/master/prow/repoowners/repoowners.go
+
+reviewers:
```

Comment:
Could you add both Nick and Xin as reviewers as well?
```
+- nlopezgi
+- xingao267
```

Reply:
lets just add nlopezgi (we'll add xingao267 later)

Reply:
Done